common/src/main/java/org/keycloak/common/util/ServerCookie.java

```diff
@@ -173,7 +187,8 @@ public static void appendCookieValue(StringBuffer headerBuf,
                                          String comment,
                                          int maxAge,
                                          boolean isSecure,
-                                         boolean httpOnly) {
+                                         boolean httpOnly,
+                                         SameSiteAttributeValue sameSite) {
         StringBuffer buf = new StringBuffer();
```

Comment:
A minor nitpick, but StringBuilder is faster than StringBuffer, plus it seems that's our preferred choice into the codebase.

Reply:
@abstractj It was like this before, I wanted to change as little things as possible so I haven't touched this. Do you want me to switch it for StringBuffer?

Reply:
@vmuzikar I overlooked this. That's not a deal breaker, we can leave it as is.